loader/src/loader.c

```diff
@@ -154,6 +154,8 @@ putc(uint8_t ch)
 {
     while (!(*UART_REG(STAT) & STAT_TDRE)) { }
     *UART_REG(TRANSMIT) = ch;
+    if (ch == '\n')
+        *UART_REG(TRANSMIT) = '\r';
```

Comment:
This will print "\n\r", but actually it should be "\r\n" that gets printed. This is a common bug in many UART drivers from seL4, we should really stop spreading it further.